src/qt/notificator.cpp

```diff
@@ -141,7 +141,7 @@ QVariant FreedesktopImage::toVariant(const QImage &img)
 
 void Notificator::notifyDBus(Class cls, const QString &title, const QString &text, const QIcon &icon, int millisTimeout)
 {
-    // https://developer.gnome.org/notification-spec/
+    // https://developer.gnome.org/documentation/tutorials/notifications.html
```

Comment:
I don't see how the new link helps with understanding the `Notificator::notifyDBus()` function code Perhaps it's best to just remove the outdated comment?